src/backend/database_migrations/versions/20220902_001655_bump_pathogen_sequence.py

```diff
@@ -0,0 +1,21 @@
+"""bump pathogen sequence
+
+Create Date: 2022-09-02 00:17:01.319857
+
+"""
+import enumtables  # noqa: F401
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "20220902_001655"
+down_revision = "20220831_150533"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute("select nextval('aspen.pathogens_id_seq')")
```

Comment:
Fix our pathogens sequence since the migration that added SC2 used an explicit ID

Reply:
oh no what did i do? i'm confused 😢

Reply:
the migration that does the insert, inserts a specific ID for the SC2 pathogen, instead of leaving that field out of the insert and letting the sequence fill in the blank. So we wound up with a sequence that's ready to insert row ID=1 and a table that already had row ID=1 in it. An example below:

![Screen Shot 2022-09-02 at 10 25 43 AM](https://user-images.githubusercontent.com/475208/188206497-bf7d5b69-b1fe-4a30-b649-0bf8a78e1218.png)

Reply:
ohhhh, thanks for explaining i have learned!